setup.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 D-Wave Systems Inc.
+# Copyright 2022-2024 D-Wave Systems Inc.
```

Comment:
Guidance from legal is just to leave it unless you totally overhaul the file. :shrug: